byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -51,23 +90,26 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> places = new ArrayList<>();
+    ArrayList<String> places = new ArrayList<String>();
     for (PlaceType location : PlaceType.values()) {
-      String place = formatLocation(location);
-      places.add(place);
+      if (!NON_INTERESTS.contains(location.toString())) {
```

Comment:
if NON_INTERESTS is always going to be matched against `PlaceType` values, consider using ImmutableSet<PlaceType> rather than ImmutableSet<String>. That way there's no mistaking what format the contents are, and you don't have to worry about values names changing. 

Note that if you change to `PlaceType`, you should use `Sets.immutableEnumSet` instead of `ImmutableSet.of`.

Reply:
Fixed